esiclient/v1/node_network.py

```diff
@@ -264,21 +264,40 @@ def take_action(self, parsed_args):
         ironic_client = self.app.client_manager.baremetal
         neutron_client = self.app.client_manager.network
 
-        node = None
-        port = None
+        current_node = ironic_client.node.get(node_uuid)
```

Comment:
Not a huge deal, but I'd keep this as `node` simply because I don't see a reason to rename this variable.